target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -168,6 +168,13 @@ glinet,gl-mifi)
 glinet,gl-x750)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	;;
+meraki,mr16)
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth0"
+	ucidef_set_led_wlan "wlan1" "WLAN1" "$boardname:green:wifi1" "phy0assoc"
+	ucidef_set_led_wlan "wlan2" "WLAN2" "$boardname:green:wifi2" "phy0assoc"
+	ucidef_set_led_wlan "wlan3" "WLAN3" "$boardname:green:wifi3" "phy0assoc"
```

Comment:
What's that supposed to do?

Reply:
https://github.com/btvmeshnet/openwrt/pull/1#pullrequestreview-451141038

Reply:
I can see that.

My question was referring to the strange situation that you have two WiFi interfaces, four wifi LEDs, and set up all of them to only one phy but three with a different trigger than the fourth one.
Despite, what's phy0assoc doing, I've never seen this so far.

Reply:
Fair. it's intended to be an RSSI bar in OEM, but I'd have used phy0rx / phy0tx / phy1rx / phy1tx for each of WLAN LEDs 1-4. 

I will have to commit and re-squash to fix this, yes? I'd be putting the new squash of commits into a different pull request, then?

Reply:
> I will have to commit and re-squash to fix this, yes? I'd be putting the new squash of commits into a different pull request, then?

Squash your changes, and force-push them here again. Don't open a new PR, we will just lose the discussion and have 10 PRs for the same subject after all.

> Fair. it's intended to be an RSSI bar in OEM, but I'd have used phy0rx / phy0tx / phy1rx / phy1tx for each of WLAN LEDs 1-4.

Just looked at the manual, yes, it's used as an RSSI bar, but we have two interfaces, so we cannot really implement this properly. Therefore, I typically prefer to have this unset, so the user can define it to his/her need.

Reply:
> Squash your changes, and force-push them here again. 

OK, great. I'll have those after work.

Reply:
I believe I have correctly reconfigured these to be empty. This awaits your review.